src/main/java/de/torui/coflsky/WSCommandHandler.java

```diff
@@ -69,17 +69,22 @@ private static void PlaySound(Command cmd, Entity sender) {
 	private static void Execute(Command cmd, Entity sender) {
 		System.out.println("Execute: " + cmd.getData() + " sender:" + sender);
 		String dummy = WSClient.gson.fromJson(cmd.getData(), String.class);
-		
-		if(dummy.startsWith("/cofl")) {
-			ClientCommandHandler.instance.executeCommand(sender, dummy);
+		Execute(dummy,sender);	
+	}
+
+	public static void Execute(String cmd, Entity sender)
+	{
+		if(cmd.startsWith("/cofl") || cmd.startsWith("http")) {
+			ClientCommandHandler.instance.executeCommand(sender, cmd);
 		} else {
-			Minecraft.getMinecraft().thePlayer.sendChatMessage(dummy);
+			Minecraft.getMinecraft().thePlayer.sendChatMessage(cmd);
 		}
-		
 	}
 
 	
 	private static IChatComponent CommandToChatComponent(WriteToChatCommand wcmd) {
+		if(wcmd.OnClick != null)
```

Comment:
I think this will cause issues.
Since if i recall correctly the last part of a `chatMessage` WSCommand is currently always a "?" with an OnClick?

Reply:
I now append a empty string with the correct click action to the end to mitigate this.